Technical-Steering-Committee/issues.md

```diff
@@ -23,27 +23,46 @@ A question about how something works in the specific projects. See the [recommen
+*Address in the next 6 - 12 months*: The submitter has indicated or confirmed that this issue is not causing significant detrimental impact - however, the technology is not functioning properly and the impact to usage could be significant. The issue should be considered and planned for an upcoming PI, provided that other higher or equivalent priority defects have not utilized available capacity.
 - **priority-low**  
-The submitter has indicated or confirmed this issue is either more cosmetic or a nice-to-have improvement. The issue should be prioritized and planned based on popularity (see voting), capacity, and submitter-availability for engagement and participation in the resolution.
+*Address in the next 12 months*: The submitter has indicated or confirmed that this defect is not causing significant detrimental impact and the impact to usage is manageable. The issue should be prioritized and planned based on popularity (see voting), capacity, and submitter-availability for engagement and participation in the resolution.
 
-For Bugs, it is also possible to reflect the perceived impact of the issue by its severity:
+For bugs, apply a severity label based on the following criteria:
 
 - **severity-critical**  
-A bug that has major impact on the organization preventing the usage of Zowe
+*Prevents Use of Zowe*: Major impact on the organization (i.e., production outage, data corruption, security exposure)
 - **severity-high**  
-A bug for which there may be workaround but limits the usage of the Zowe for major use cases
+*Limits Use of Zowe / no (or limited) workaround*: Large impact to usage, workaround may exist but limits usage
 - **severity-medium**  
-A bug where workaround exists or that does not prevent the usage of the Zowe or the use case, just makes it more complex
+*Limits Use of Zowe / workaround*: Large impact to usage, workaround fixes the defect but makes usage more complex 
 - **severity-low**  
-A bug that makes Zowe usage less convenient but does not impact key use cases
+*Uncomfortable Use of Zowe*: Low impact to usage, does not impact key use cases, workaround may exist
+
+## Triaging Guidelines: *Enhancements*
+
+Importance of the issue as perceived by squad:
+
+- **priority-critical**  
+*Address Immediately*: The submitter has indicated (or the reviewer has judged) that the enhancement is necessary to PREVENT significant detrimental impact to their usage and or their system which may result in production outage, data corruption, security exposure, etc. The enhancement should be pulled in immediately to be addressed. If this is not possible, the enhancement should be escalated to the TSC immediately. Submitter should commit to engagement throughout the development and validation of the enhancement (exceptions may apply).
```

Comment:
Could you provide an example of such an enhancement?

Reply:
This could apply to work done to replace a dependency that is going to become EOL, or it could be some other software change that is necessary to avoid problems (that could have financial/security consequences) in the near future. There may not be many opportunities to classify an enhancement as priority-critical, but I think it is useful to have a priority above priority-high to identify enhancements that should be addressed/implemented immediately vs. enhancements that should be addressed some time in the current PI or the next PI.

Also, as a reminder, the vote is still active in the tsc channel if you would like to cast a vote.

Reply:
Thank for the details. What about the following
> Also, should the submitter assign a priority or should the team do that based on the criticality and the team's other work?